### PR TITLE
Feature/implement authentication on backend

### PR DESCRIPTION
**Pull request by viviviviviivilll**
Auth 1.0

### CODE REVIEW

File: BobManager.API/BobManager.Domain/Services/AccountService.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            {
+                Email = entity.Email,
+                UserName = entity.Email
+            };
+            var result = await _userManager.CreateAsync(user, entity.Password);
+            if (result.Succeeded)
+            {
+               return  await Login(new LoginDto { Email = entity.Email, IsRemember = false, Password = entity.Password});
+            }
+            return _clientErrorManager.MapErrorIDToResultDto(2); ;
```
Comment (Firto) [inline BobManager.API/BobManager.Domain/Services/AccountService.cs]: Remove braces

**Review by ander21 — APPROVED** [review]
Approve

**Review by Firto — APPROVED** [review]
Approve

**Review by NerevitVasyan — APPROVED** [review]
Approve

**Pull request merged**